shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/execute/PipelineJobExecutor.java

```diff
@@ -81,9 +63,18 @@ private static void dispatchEvent(final DataChangedEvent event) {
                     log.error("analyze job config pojo failed.", ex);
                     return;
                 }
-                entry.getValue().handler(event, jobConfigPOJO);
+                entry.getValue().handle(event, jobConfigPOJO);
                 return;
             }
         }
     }
+    
+    /**
+     * Get pipeline job executor instance.
+     *
+     * @return pipeline job executor
+     */
+    public static PipelineJobExecutor getInstance() {
+        return INSTANCE;
+    }
```

Comment:
Could be put after `INSTANCE`

Reply:
Can't put it after `INSTANCE `, because must after `private Constructor`